Header header
string arm_id             # ID of the robot arm this trajectory is for
float64 execution_time    # Total time to execute the trajectory in seconds
TrajectoryPoint[] points  # Sequence of trajectory points 